src/main/java/kakao/rebit/diary/service/DiaryService.java

```diff
@@ -0,0 +1,81 @@
+package kakao.rebit.diary.service;
+
+import kakao.rebit.book.entity.Book;
+import kakao.rebit.book.repository.BookRepository;
+import kakao.rebit.diary.dto.DiaryRequest;
+import kakao.rebit.diary.dto.DiaryResponse;
+import kakao.rebit.diary.entity.Diary;
+import kakao.rebit.diary.repository.DiaryRepository;
+import kakao.rebit.member.entity.Member;
+import kakao.rebit.member.service.MemberService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class DiaryService {
+
+    private final DiaryRepository diaryRepository;
+    private final MemberService memberService;
+    private final BookRepository bookRepository;
+
+    public DiaryService(DiaryRepository diaryRepository, MemberService memberService,
+        BookRepository bookRepository) {
+        this.diaryRepository = diaryRepository;
+        this.memberService = memberService;
+        this.bookRepository = bookRepository;
+    }
+
+    @Transactional(readOnly = true)
+    public Page<DiaryResponse> getDiaries(Long memberId, Pageable pageable) {
+        return diaryRepository.findByMemberId(memberId, pageable)
+            .map(diary -> new DiaryResponse(diary.getId(), diary.getContent(),
+                diary.getMember().getId(),
+                diary.getBook().getIsbn()));
+    }
+
+    @Transactional(readOnly = true)
+    public DiaryResponse getDiaryById(Long memberId, Long id) {
+        Diary diary = diaryRepository.findByIdAndMemberId(id, memberId)
+            .orElseThrow(() -> new IllegalArgumentException(
+                "회원 ID " + memberId + "에 해당하는 다이어리 ID " + id + "를 찾을 수 없습니다."));
```

Comment:
```suggestion
                String.format("회원 ID %d 에 해당하는 다이어리 ID %d 를 찾을 수 없습니다.", memberId, id)
```
문자열을 이런식으로도 표현할 수 있어요~ 
밑에 64 Line 에도 똑같은 문자열이 있네요. 에러메시지 양식에 변경이 생기면 둘다 수정을 해줘야 해요. 공통으로 사용되는 문자열은 변수로 빼면 어떨까요? enum 이나 별도로 에러메시지를 관리하는 클래스를 만들수도 있을 것 같아요. :)